client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -39,46 +39,56 @@ const RegisteredDomain = React.createClass( {
 		);
 	},
 
-	getPrivacyProtection() {
-		if ( this.props.domain.hasPrivacyProtection ) {
-			const path = paths.domainManagementContactsPrivacy(
-				this.props.selectedSite.domain,
-				this.props.domain.name
-			);
-
-			return (
-				<a href={ path }>
-					<Notice
-						isCompact
-						status="is-success"
-						icon="lock">
-						{ this.translate( 'On', {
-							context: 'An icon label when Privacy Protection is enabled.'
-						} ) }
-					</Notice>
-				</a>
-			);
-		}
-
-		const path = paths.domainManagementPrivacyProtection(
-			this.props.selectedSite.domain,
-			this.props.domain.name
-		);
-
+	getLabel( { status, icon, message, href } ) {
 		return (
-			<a href={ path } onClick={ this.recordEvent( 'noneClick', this.props.domain ) }>
+			<a href={ href }>
```

Comment:
Maybe instead of hacking together a clickable `Notice`, a compact `Button` would be a better fit here? It looks more "clickable" too :)
I know that's how it's been before this PR, but might as well improve it when we can. Or is this a deliberate design decision? @breezyskies? :grin: